tf-modules/cumulus-rds-tf/main.tf

```diff
@@ -68,11 +82,14 @@ resource "aws_rds_cluster" "cumulus" {
   scaling_configuration {
     max_capacity = var.max_capacity
     min_capacity = var.min_capacity
+    timeout_action = var.rds_scaling_timeout_action
```

Comment:
I also wanted to specify the behavior for our serverless autoscaling timeout, but Terraform doesn't support that yet:

https://github.com/hashicorp/terraform-provider-aws/issues/22292